packages/hint-compat-api/src/helpers/compat-css.ts

```diff
@@ -0,0 +1,285 @@
+    }
+
+    public async reportError(featureName: string, message: string, location?: ProblemLocation): Promise<void> {
+        this.cachedFeatures.addError(featureName, this.hintResource, message, location);
+        await this.hintContext.report(this.hintResource, null, message, featureName, location);
+    }
+
+    public async reportIfThereIsNoInformationAboutCompatibility(message: string, browsersToSupport: BrowserSupportCollection, browserToSupportName: string, featureName: string, location?: ProblemLocation): Promise<void> {
+        if (!this.wasBrowserSupportedInSometime(browsersToSupport, browserToSupportName) && Object.keys(browsersToSupport).includes(browserToSupportName)) {
+            await this.reportError(featureName, message, location);
+        }
+    }
+
+    public wasBrowserSupportedInSometime(browsersToSupport: BrowserSupportCollection, browserToSupportName: string): boolean {
+        return Object.entries(browsersToSupport).some(([browserName]) => {
+            if (browserName !== browserToSupportName) {
+                return false;
+            }
+
+            return true;
+        });
+    }
+
+    public generateNotSupportedVersionsError(featureName: string, notSupportedVersions: string[], statusName: string, prefix?: string): string {
+        const usedPrefix = prefix ? `prefixed with ${prefix} ` : '';
+        const groupedNotSupportedVersions = this.groupNotSupportedVersions(notSupportedVersions);
+
+        return `${featureName} ${usedPrefix ? usedPrefix : ''}is not ${statusName} on ${groupedNotSupportedVersions.join(', ')} browser${notSupportedVersions.length > 1 ? 's' : ''}.`;
+    }
+
+    /**
+     * @method groupNotSupportedVersions
+     * Examples:
+     * [ 'chrome 66', 'chrome 69' ] into ['chrome 66, 69']
+     * [ 'chrome 67', 'chrome 68', 'chrome 69' ] into ['chrome 67-69']
+     * [ 'chrome 66', 'chrome 68', 'chrome 69' ] into ['chrome 66, 67-69']
+     *
+     */
+    private groupNotSupportedVersions(notSupportedVersions: string[]): string[] {
+        if (!notSupportedVersions) {
+            return [];
+        }
+
+        const browsers: BrowserVersions = {};
+
+        notSupportedVersions.forEach((browserAndVersion: string) => {
+            const [browser, version] = browserAndVersion.split(' ');
+
+            browsers[browser] = browsers[browser] || [];
+            browsers[browser].push(version);
+        });
+
+        const groupedVersions = Object.entries(browsers).map(([browser, versions]) => {
+            const sortedVersions = versions.sort();
+            let grouped = '';
+            let groupStarted = false;
+
+            sortedVersions.forEach((value, i) => {
+                const nextValue = sortedVersions[i + 1];
+
+                if (!groupStarted) {
+                    grouped += `${browser} ${value}`;
+                }
+
+                if (nextValue && Number(nextValue) - Number(value) > 1) {
```

Comment:
value and nextValue is not a number by default?

Reply:
Can be something like 4.4.4 https://github.com/mdn/browser-compat-data/issues/3034
![image](https://i.imgur.com/vCsjPRy.png)

But I've updated to be safe against '4.4.4' because we forgot, thanks.